packages/mdc-card/README.md

```diff
@@ -39,200 +39,146 @@ npm install --save @material/card
+should use the `mdc-card__action--icon` class.
 
-Media items are designed to be used in horizontal blocks, taking up a fixed height, rather than stretching to the width
-of the card.
-
-> Note: Unlike media blocks, media items are not designed to be used as optional containers.
+To have a single action button take up the entire width of the action row, use the `--full-bleed` modifier:
 
 ```html
-<div class="mdc-card__horizontal-block">
-  <section class="mdc-card__primary">
-    <h1 class="mdc-card__title mdc-card__title--large">Title here</h1>
-    <h2 class="mdc-card__subtitle">Subtitle here</h2>
-  </section>
-  <img class="mdc-card__media-item mdc-card__media-item--1x" src="image.jpg">
+<div class="mdc-card__actions mdc-card__actions--full-bleed">
+  <a class="mdc-button mdc-card__action" href="javascript:">
+    <span>All Business Headlines</span>
+    <i class="material-icons">arrow_forward</i>
+  </a>
 </div>
 ```
 
-There are several predefined media item sizes you can use:
-
-| Class                           | Description                                                      |
-| ------------------------------- | ---------------------------------------------------------------- |
-| `mdc-card__media-item`          | Defines a media item and sets it to the default height of 80px.  |
-| `mdc-card__media-item--1dot5x`  | Sets the media item height to 120px.                             |
-| `mdc-card__media-item--2x`      | Sets the media item height to 160px.                             |
-| `mdc-card__media-item--3x`      | Sets the media item height to 240px.                             |
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-card` | A card
+`mdc-card__media` | Media area that displays a custom `background-image` with `background-size: cover`
+`mdc-card__media--square` | Automatically scales the media area's height to equal its width
+`mdc-card__media--16-9` | Automatically scales the media area's height according to its width, maintaining a 16:9 aspect ratio
+`mdc-card__media-content` | An absolutely-positioned box the same size as the media area, for displaying a title or icon on top of the `background-image`
+`mdc-card__actions` | Row containing action buttons and/or icons
+`mdc-card__actions--full-bleed` | Removes the action area's padding and causes its only child (an `mdc-card__action` element) to consume 100% of the action area's width
+`mdc-card__action-buttons` | A group of action buttons, displayed on the left side of the card (in LTR), adjacent to `mdc-card__action-icons`
+`mdc-card__action-icons` | A group of supplemental action icons, displayed on the right side of the card (in LTR), adjacent to `__action-buttons`
+`mdc-card__action` | An individual action button or icon
+`mdc-card__action--icon` | An icon action
+
+### Sass Mixins
+
+Mixin | Description
+--- | ---
+`mdc-card-fill-color($color)` | Sets the fill color of a `mdc-card`
```

Comment:
Just say card instead of `mdc-card` here, like you do on the next line?

Reply:
Done